release_notes.md

```diff
@@ -0,0 +1,6 @@
+## Release Notes
```

Comment:
Release notes are of the previous version, Also the folder structure of release notes is different